recipes/lld/meta.yaml

```diff
@@ -40,7 +40,6 @@ test:
 
   commands:
     - ld.lld --version  # [linux]
-    - ld --version  # [osx]
```

Comment:
Why?

Reply:
That command failed and I don't have access to an osx machine to are which command would work.